includes/Checker/Abstract_Check_Runner.php

```diff
@@ -270,11 +270,12 @@ final public function get_checks_to_run() {
 		// Include file to use is_plugin_active() in CLI context.
 		require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
+		$runtime     = new Runtime_Environment_Setup();
 		$check_slugs = $this->get_check_slugs();
 		$check_flags = Check_Repository::TYPE_STATIC;
 
 		// Check if conditions are met in order to perform Runtime Checks.
-		if ( $this->initialized_early && is_plugin_active( $this->get_plugin_basename() ) ) {
+		if ( $runtime->can_setup_runtime_environment() && is_plugin_active( $this->get_plugin_basename() ) ) {
```

Comment:
I think this needs to check if _either_ the runner was initialized early _or_ the runtime environment can be set up. If it was initialized early, we know the runtime environment is already set up, so no need to check for it.
```suggestion
		if ( ( $this->initialized_early || $runtime->can_setup_runtime_environment() ) && is_plugin_active( $this->get_plugin_basename() ) ) {
```

Reply:
Thanks @felixarntz updated